### PR TITLE
Second attempt to create notification channel if the network isn't connected

### PR DESCRIPTION
**Pull request by MatkovIvan**

### CODE REVIEW

**Comment by MatkovIvan** [comment]
`error : Could not create the Java Virtual Machine.`
Try again...

**Comment by guperrot** [comment]
Looks good but needs @achocron review as it modifies the core and not sure the network adapter modification would be okay with him as is.

File: SDK/AppCenterPush/Microsoft.AppCenter.Push.UWP/Push.cs [hunk 1]
```diff
@@ -141,6 +141,53 @@ private void OnPushNotificationReceivedHandler(PushNotificationChannel sender, W
...
+        private async Task<PushNotificationChannel> CreatePushNotificationChannelAsync()
+        {
+            PushNotificationChannel channel = null;
+            try
+            {
+                // If this isn't the first time after installation, the notification channel is created successfully even without network.
+                channel = await new WindowsPushNotificationChannelManager().CreatePushNotificationChannelForApplicationAsync()
+                    .AsTask().ConfigureAwait(false);
+            }
+            catch (Exception exception)
```
Comment (achocron) [inline SDK/AppCenterPush/Microsoft.AppCenter.Push.UWP/Push.cs]: Is it necessary to catch any exception type, or can you make this more specific?

Comment (MatkovIvan) [reply SDK/AppCenterPush/Microsoft.AppCenter.Push.UWP/Push.cs]: Yes, `System.Exception` should be caught.
Sample one:
```C#
System.Exception occurred
  HResult=0x800704C6
  Message=The network is not present or not started. (Exception from HRESULT: 0x800704C6)
  Source=<Cannot evaluate the exception source>
  StackTrace:
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.ConfiguredTaskAwaitable`1.ConfiguredTaskAwaiter.GetResult()
   at Microsoft.AppCenter.Push.Push.<CreatePushNotificationChannelAsync>d__7.MoveNext() in SDK\AppCenterPush\Microsoft.AppCenter.Push.UWP\Push.cs:line 150
```

**Pull request merged**